daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Profile.scala

```diff
@@ -92,9 +91,7 @@ object Profile {
     ) {
       def write(path: Path): Unit = {
         import JsonProtocol.fileFormat
-        val writer = new BufferedWriter(new FileWriter(path.toFile()))
-        writer.write(this.toJson.compactPrint)
-        writer.close()
+        val _ = Files.write(path, this.toJson.compactPrint.getBytes)
```

Comment:
@S11001001 Does `spray-json` provide a way to write the JSON object into a file without first converting it to a string? We're talking about fairly big JSON objects of multiple MB here.

Reply:
I had a quick look into it and couldn't find anything that returned a `Writer` or `OutputStream`, unfortunately.